src/session-refresh-delegate.ts

```diff
@@ -63,6 +63,8 @@ export class SessionRefreshDelegate<S extends Schema> {
       });
       await this._refreshPromise;
       this._logger.info('<refresh complete>');
+    } catch(err) {
+      throw new Error(`Unable to refresh session due to: ${(err as Error).message}`)
```

Comment:
the error passed in the callback is typed, it's guaranteed to be an instance of `Error`:
https://github.com/jsforce/jsforce/blob/3c9e4e6c658a3660c5038c53745283215313138a/src/types/common.ts#L9